mne_bids/path.py

```diff
@@ -571,9 +571,11 @@ def get_kinds(bids_root):
     return kinds
 
 
-def get_entity_vals(bids_root, entity_key, *, ignore_sub='emptyroom',
-                    ignore_task=None, ignore_ses=None, ignore_run=None,
-                    ignore_acq=None):
+def get_entity_vals(bids_root, entity_key, *, ignore_subjects='emptyroom',
+                    ignore_sessions=None, ignore_tasks=None, ignore_runs=None,
+                    ignore_processings=None, ignore_spaces=None,
+                    ignore_acquisitions=None, ignore_splits=None,
+                    ignore_kinds=None):
```

Comment:
this is kind of a backwards-incompatible change but I guess that's fine

Reply:
Yes… I hope so :D